### PR TITLE
User information updated.

### PR DESCRIPTION
**Pull request by AtefR**
Update user information to get domain, tax and commercial numbers.

Current [user information response](https://docs.salla.dev/docs/merchant/61d8f18097b4e-user-information):

```
{
  "status": 200,
  "success": true,
  "data": {
    "id": 1689171978,
    "name": "Test User",
    "email": "test@gmail.com",
    "mobile": "+966555555555",
    "role": "user",
    "created_at": "2021-03-27 21:51:56",
    "merchant": {
      "id": 847769313,
      "username": "User_name123",
      "name": "User Name",
      "avatar": "https://i.ibb.co/jyqRQfQ/avatar-male.webp",
      "store_location": "21.589481104199123,39.67869125586653",
      "plan": "pro",
      "status": "active",
      "domain": "https://www.domain.com",
      "tax_number": "424243241321234",
      "commercial_number": "3552100509",
      "created_at": "2021-12-31 12:59:59"
    }
  }
}
```

### CODE REVIEW

File: src/Provider/SallaUser.php [hunk 1]
```diff
@@ -168,6 +168,36 @@ public function getStoreStatus()
...
+     *
+     * @return string|null
+     */
+    public function getStoreTaxNumber()
+    {
+        return $this->getResponseValue('data.merchant.tax_number');
+    }
+
+    /**
+     * Get store status.
```
Comment (salkhwlani) [inline src/Provider/SallaUser.php:192]: update the docs

File: src/Provider/SallaUser.php [hunk 2]
```diff
@@ -168,6 +168,36 @@ public function getStoreStatus()
...
+     *
+     * @return string|null
+     */
+    public function getStoreCommercialNumber()
+    {
+        return $this->getResponseValue('data.merchant.tax_number');
```
Comment (salkhwlani) [inline src/Provider/SallaUser.php:198]: i think this should be `commercial_number`

File: src/Provider/SallaUser.php [hunk 3]
```diff
@@ -168,6 +168,36 @@ public function getStoreStatus()
         return $this->getResponseValue('data.merchant.status');
     }
 
+    /**
+     * Get store status.
+     *
+     * @return string|null
+     */
+    public function getStoreDomain()
```
Comment (salkhwlani) [inline src/Provider/SallaUser.php:176]: remove this, is already supported in the master

**Review by salkhwlani — CHANGES_REQUESTED** [review]

File: src/Provider/SallaUser.php [hunk 4]
```diff
@@ -167,9 +167,9 @@ public function getStoreStatus()
     {
         return $this->getResponseValue('data.merchant.status');
     }
-    
-     /**
-     * Get store domain.
+
+    /**
+     * Get store status.
```
Comment (HazemKhaled) [inline src/Provider/SallaUser.php:172]:
```suggestion
     * Get store domain.
```

**Review by HazemKhaled — APPROVED** [review]

**Review by salkhwlani — APPROVED** [review]

**Pull request merged**